src/jquery/css.js

```diff
@@ -81,10 +81,33 @@ if ( jQueryVersionSince( "3.4.0" ) && typeof Proxy !== "undefined" ) {
 	} );
 }
 
-// Create a dummy jQuery.cssNumber if missing. It won't be used by jQuery but
-// it will prevent code adding new keys to it unconditionally from crashing.
+// In jQuery >=4 where jQuery.cssNumber is missing fill it with the latest 3.x version:
+// https://github.com/jquery/jquery/blob/3.6.0/src/css.js#L212-L233
+// This way, number values for the CSS properties below won't start triggering
+// Migrate warnings when jQuery gets updated to >=4.0.0 (gh-438).
 if ( !jQuery.cssNumber ) {
-	jQuery.cssNumber = {};
+	jQuery.cssNumber = {
```

Comment:
Man we can never win. 😸

Reply:
This is not 100% necessary as Core doesn't need this object in 4.x; we could as well just enumerate allowlisted properties in Migrate. However, by doing it via `jQuery.cssNumber` we have the same Migrate code path for all jQuery versions and it's easier to copy the object from the `3.x-stable` Core branch if there are any changes.

Reply:
Yeah it seems easier to just copy over the list. It's an exposed object so who knows, maybe some software out there is looking at it and not just setting properties.